solutions/bronze/dmoj-spirale.mdx

```diff
@@ -73,3 +73,64 @@ int main() {
 	}
 }
 ```
+
+</CPPSection>
+<PySection>
+
+```py
+ux = [0, -1, 0, 1]
+uy = [1, 0, -1, 0]
+dx = [0, -1, 0, 1]
+dy = [-1, 0, 1, 0]
+
+mat = [[(float("inf"))] * 50 for _ in range(50)]
```

Comment:
magic #

Reply:
added

Reply:
AC: https://dmoj.ca/submission/6718196

Reply:
> AC: https://dmoj.ca/submission/6718196

fyi dmoj submissions are private unless you solved the problem